client/src/components/UserProfile.js

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+
+const UserProfile = (props) => {
+  let user = {username:'', email:'', userId:''} 
```

Comment:
this is placeholder information, delete it.

Reply:
we should be able to get this page to displace without having to make this empty object.

Reply:
In order to remove the empty object, we are finding that we have to create a new variable and set it to null in order to render properly on initial page load. This is resulting in more code than with the empty object. Is that still preferable?